front/src/applications/operationalStudies/components/Scenario/Timetable.jsx

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { FaPlus } from 'react-icons/fa';
+import PropTypes from 'prop-types';
+
+import nextId from 'react-id-generator';
+import InputSNCF from 'common/BootstrapSNCF/InputSNCF';
+import { useDebounce } from 'utils/helpers';
+import { trainscheduleURI } from 'applications/operationalStudies/components/SimulationResults/simulationResultsConsts';
+import {
+  updateMustRedraw,
+  updateSelectedProjection,
+  updateSelectedTrain,
+} from 'reducers/osrdsimulation/actions';
+import { deleteRequest, get, post } from 'common/requests';
+import { setFailure, setSuccess } from 'reducers/main';
+import trainNameWithNum from 'applications/operationalStudies/components/ManageTrainSchedule/AddTrainSchedule/trainNameHelper';
+import { MANAGE_TRAIN_SCHEDULE_TYPES } from 'applications/operationalStudies/consts';
+import getTimetable from './getTimetable';
+import TimetableTrainCard from './TimetableTrainCard';
+
+/* function trainsDurations(trainList) {
+  const durationList = trainList.map((train) => ({
+    id: train.id,
+    duration:
+      train.arrival > train.departure
+        ? train.arrival - train.departure
+        : train.arrival + 86400 - train.departure,
+  }));
+  const min = Math.min(...durationList.map((train) => train.duration));
+  const max = Math.max(...durationList.map((train) => train.duration));
+  // console.log(durationList.map((train) => train.duration), min, max);
+  return durationList;
+} */
+
```

Comment:
logs

Reply:
It's work for futur, will be kept until fully implemented